src/github.com/couchbase/sync_gateway/base/bucket_gocb.go

```diff
@@ -20,14 +20,22 @@ import (
 
 var gocbExpvars *expvar.Map
 
+const (
+	MaxConcurrentSingleOps = 1000 // Max 1000 concurrent single bucket ops
+	MaxConcurrentBulkOps   = 35   // Max 35 concurrent bulk ops
+	MaxBulkSetSize         = 100  // Maximum number of ops per bulk call
```

Comment:
The variable name doesn't match the comment -- is this specific to BulkSet or for any bulk op?